webapp/src/client/client.js

```diff
@@ -5,6 +5,7 @@ import AuthenticationContext from 'adal-angular';
 import {isDesktopApp} from '../utils/user_utils';
 import {Periods} from '../constants';
 import {id as pluginID} from '../manifest';
+import {getPluginServerRoute} from 'selectors';
```

Comment:
Can we make this a relative import `../selectors` like the other local imports here?

Reply:
Removed